### PR TITLE
Update inbound request pipeline

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Don't know for sure but think it belongs to: https://github.com/umbraco/UmbracoDocs/issues/1873

Update inbound pipeline for v8

PublishedContentRequest --> PublishedRequest
Domain  --> DomainAndUri (and update description)
Removed RenderingEngine  since it's only MVC
Update how to subscribe 
Removed `needsV8Update`

### CODE REVIEW

**Comment by sofietoft** [comment]
This is great @erikjanwestendorp ! 😄 Thanks a lot for putting in the work for this PR.

I'll make sure we have it reviewed as soon as possible 💪

File: Reference/Routing/Request-Pipeline/inbound-pipeline.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+versionFrom: 8.0.0
 ---
 
 # Inbound request pipeline
 
-The inbound process is triggered by the Umbraco (http) Module.  The **[published content request preparation](published-content-request-preparation.md)** process kicks in and creates a `PublishedContentRequest`.
+The inbound process is triggered by the Umbraco (http) Module.  The **[published content request preparation](published-content-request-preparation.md)** process kicks in and creates a `PublishedRequest`.
 
-The `PublishedContentRequest` object represents the request which Umbraco must handle.  It contains everything that will be needed to render it.  All this happens when the Umbraco modules thinks it's a document to render.
+The `PublishedRequest` object represents the request which Umbraco must handle.  It contains everything that will be needed to render it.  All this happens when the Umbraco modules thinks it's a document to render.
```
Comment (marcemarc) [inline Reference/Routing/Request-Pipeline/inbound-pipeline.md]:
```suggestion
The `PublishedRequest` object represents the request which Umbraco must handle.  It contains everything that will be needed to render it.  All this occurs when the Umbraco modules consider that an incoming request maps to a document that can be rendered.
```

File: Reference/Routing/Request-Pipeline/inbound-pipeline.md [hunk 2]
```diff
@@ -21,12 +20,10 @@ There are 4 important properties, which contains all the information to find bac
...
 ```csharp
 public bool HasDomain { get; }
-public Domain Domain { get; }
-public Uri DomainUri { get; }
+public DomainAndUri Domain { get; }
 public CultureInfo Culture { get; }
 ```
-
-Domain contains: "example.com", while Uri is "http://example.com".
+Domain is a DomainAndUri object ie a standard Domain plus the fully qualified uri. For example, the Domain may contain "example.com" whereas the Uri will be fully qualified eg "http://example.com/".
```
Comment (marcemarc) [inline Reference/Routing/Request-Pipeline/inbound-pipeline.md]:
```suggestion
Domain is a DomainAndUri object ie a standard Domain plus the fully qualified uri. For example, the Domain may contain "example.com" whereas the Uri will be fully qualified eg "https://example.com/".
```

File: Reference/Routing/Request-Pipeline/inbound-pipeline.md [hunk 3]
```diff
@@ -21,12 +20,10 @@ There are 4 important properties, which contains all the information to find bac
...
 
 It contains also the content to render:
```
Comment (marcemarc) [inline Reference/Routing/Request-Pipeline/inbound-pipeline.md]:
```suggestion
It contains the content to render:
```

File: Reference/Routing/Request-Pipeline/inbound-pipeline.md [hunk 4]
```diff
@@ -45,19 +42,30 @@ Contains template information and the corresponding rendering engine:
 ```csharp
 public bool HasTemplate { get; }
 public string TemplateAlias { get; }
-public RenderingEngine RenderingEngine { get; }
 public bool TrySetTemplate(string alias);
 public void SetTemplate(ITemplate template);
 ```
 
-You can subscribe to the event to know when the `PublishedContentRequest` is ready to be processed.  It's up to you to change anything (content, template, ...):
+You can subscribe to the event to know when the `PublishedRequest` is ready to be processed.  It's up to you to change anything (content, template, ...):
```
Comment (marcemarc) [inline Reference/Routing/Request-Pipeline/inbound-pipeline.md]:
```suggestion
You can subscribe to the 'Prepared' event which is triggered just after the point when the`PublishedRequest` is prepared - (but before it is ready to be processed).  Here modify anything in the request before it is processed!  eg. content, template, etc:
```

**Comment by erikjanwestendorp** [comment]
@marcemarc Thanks for reviewing! 👍 I updated the the PR with your suggestions :-)

**Comment by marcemarc** [comment]
@erikjanwestendorp thanks again, it's great to get these more complex docs updated, with the right class names after the V8 refactoring! and @sofietoft yes I have merged a PR with a Vale warning... but it was warning against the use of the word 'just' but here it is used correctly describing something 'just before' rather than the bad use you 'just' do this then 'just' do that.... ok...

**Pull request merged**